AmericanMedicalAssociation/AMA-style-guide
# Feature/ewl 3970 modify related content

## Ticket(s)

**Jira Ticket**

- [EWL-3970: Topics |SG| Reconcile Related Content Styles](https://issues.ama-assn.org/browse/EWL-3970)


## Description

Removes old topic related content patterns. 
Creates a topic related content molecule with an "with image" variant and leverages pseudo patterns. 
Creates a topic related content list organism that pulls in both with and without image variants.


## To Test

- [ ] Review Molecules > Components > Topics Related Content pattern
- [ ] Review Molecules > Components > Topics Related Content With Image pattern
- [ ] Review Molecules > Organisms > Topic > Topic Related Content List pattern
- [ ] Review Topics template

See that these all look reasonable.
See that there aren't any other topics related content patterns floating around in the styleguide.

## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
Ticket [EWL-3970: Topics |SG| Reconcile Related Content Styles](https://issues.ama-assn.org/browse/EWL-3970) also covers Drupal implementation so please only update this ticket when merged, do not move it to testing.


## Additional Notes

Waiting for @r-kitchen's go-ahead on this work.
